src/main/java/com/xceptance/neodymium/module/statement/testdata/TestdataStatement.java

```diff
@@ -191,20 +191,18 @@ else if (!classDataSetAnnotations.isEmpty())
             dataSetAnnotations = classDataSetAnnotations;
         }
 
+        // we've gathered some instructions to override
```

Comment:
please make this comment more clear or put it to the correct place

Reply:
Comment removed